collector/filesystem_freebsd.go

```diff
@@ -21,78 +21,65 @@ import (
 import "C"
 
 const (
-	filesystemSubsystem = "filesystem"
+	subsystem = "filesystem"
```

Comment:
They do have local scope. Just that in Go, the file has no own scope. The local scope here is the package.